geo-types/src/macros.rs

```diff
@@ -284,3 +284,82 @@ macro_rules! polygon {
         )
     };
 }
+
+#[cfg(test)]
+mod test {
+    #[test]
+    fn test_point() {
+        let p = point!(x: 1.2, y: 3.4);
+        assert_eq!(p.x(), 1.2);
+        assert_eq!(p.y(), 3.4);
+
+        let p = point! {
+            x: 1.2,
+            y: 3.4,
+        };
+        assert_eq!(p.x(), 1.2);
+        assert_eq!(p.y(), 3.4);
+    }
+
+    #[test]
+    fn test_line() {
+        let ls = line_string![(x: -1.2f32, y: 3.4f32)];
+        assert_eq!(ls[0], coord! {x: -1.2, y: 3.4});
```

Comment:
I think these should be formatted as:

```suggestion
        assert_eq!(ls[0], coord! { x: -1.2, y: 3.4 });
```

Regarding your proposal on the GeoRust Discord, this is just one way in which macros are still worse than normal Rust code. I'm strongly against deprecating the other constructors and conversions.